Source/Core/Core/GeckoCodeConfig.cpp

```diff
@@ -251,4 +250,21 @@ void SaveCodes(IniFile& inifile, const std::vector<GeckoCode>& gcodes)
   inifile.SetLines("Gecko_Enabled", enabled_lines);
   inifile.SetLines("Gecko_Disabled", disabled_lines);
 }
+
+bool DeserializeLine(const std::string& line, GeckoCode::Code* out)
+{
+  std::vector<std::string> pieces = SplitString(line, ' ');
+  bool good = false;
+
+  if (pieces.size() > 0)
+    good &= TryParse(pieces[0], &out->address, 16);
+
+  if (pieces.size() > 1)
+    good &= TryParse(pieces[1], &out->data, 16);
```

Comment:
This looks fishy. `false & anything` will never become `true`.

Reply:
Oops, yes. It should start off being `true`.